wellcomeml/viz/visualize_clusters.py

```diff
@@ -150,6 +161,7 @@ def visualize_clusters(clustering, filter_list, radius: float = 0.05,
     if output_in_notebook:
         output_notebook()
         show(column(dropdown, p))
-    else:
+
+    if output_file_path:
```

Comment:
Ok so now when setting `output_in_notebook=False` and when not specifying the `output_file_path` there's no output. I guess because of line 21: `output_file_path: Optional[str] = None`

Reply:
When setting `output_in_notebook=True, output_file_path="test.html"` it plots two times in the notebook and once on a new page